pom.xml

```diff
@@ -47,6 +47,9 @@
 
         <!-- Remove when parent is updated -->
         <dep.otj-logging.version>3.0.2</dep.otj-logging.version>
+
+        <!-- Remove when otj-parent is updated -->
```

Comment:
No, more accurately when the other is cut. We don't publish snapshots

Reply:
Yes _and_ also after the other is cut we will still need to update this once the parent is updated. Was just thinking ahead.